pkgs/by-name/ca/catppuccin-gtk-theme/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gtk-engine-murrine,
+  jdupes,
+  sassc,
+  accent ? ["default"],
+  shade ? "dark",
+  size ? "standard",
+}: let
+  validAccents = ["default" "purple" "pink" "red" "orange" "yellow" "green" "teal" "grey"];
+  validShades = ["light" "dark"];
+  validSizes = ["standard" "compact"];
+
+  single = x: lib.optional (x != null) x;
+  pname = "Catppuccin";
```

Comment:
```suggestion
  pname = "catppuccin-gtk-theme";
```

Reply:
Not sure if we want to stick with this or go with somethings else to distinguish

Reply:
I should've named it that from the start, good catch!
+ With `--name ${pname}` in mind, do you reckon having the themes in a similar format to this `catppucin-gtk-theme-pink-dark` a good idea or should I just reduce it to Catppuccin? (I think the later is cleaner, not sure though)

Reply:
Perhaps `magnetic-catppuccin-gtk` since its based of Magnetic

Reply:
Also the output theme name contains capital letters, `catppuccin-gtk-theme-Dark`, I wonder if going with lowercase `pname` + the generated capital letter shade variant is a good choice.

Reply:
@isabelroses I assume Magnetic is a theme, right? But that would be more confusing, because not everyone would know that bit of information.

Reply:
Does the latest change satisfy the `pname` requirements now?

Reply:
Pretty much conveyed by the description and the fact `gtk` is still in the name but not `catppuccin-gtk` since that was already taken and would be more confusing. We have a similar thing with `catppuccin-sddm` and `catppuccin-sddm-corners` https://search.nixos.org/packages?channel=unstable&query=catppuccin-sddm

Reply:
2nd question: since we have a couple of other themes from Fausto-Korpsvart that follows a similar format, does that mean that we have to rename them all to `magnetic-x-gtk`? I guess I can change it to `magnetic-catppuccin-gtk` if that is what y'all want. (even though I don't know what magnetic stands for still...)

Reply:
i got the magnetic from the readme since its based of some `magnetic` gtk theme https://github.com/Fausto-Korpsvart/Catppuccin-GTK-Theme/blob/0bd2869e7f0fdb36c720a4fb873d4fed361b0606/README.md?plain=1#L11

> rename them all to `magnetic-x-gtk`

I don't think it's that important, this is just because of the name clash and keeping them seperated.

Reply:
Alright! Just renamed the PR + commit with changes to the commited file. Please take a look at the updated version! :blush: